pkg/workloadmanager/handlers.go

```diff
@@ -183,8 +193,55 @@ func (s *Server) handleCreateSandbox(c *gin.Context) {
 		return
 	}
 
-	// TODO: Code Interpreter sandbox created, init code interpreter
-	return
+	// Code Interpreter sandbox created, init code interpreter
+	// Find the /init endpoint from entryPoints
+	var initEndpoint string
+	for _, access := range redisCacheInfo.EntryPoints {
+		if access.Path == "/init" {
+			initEndpoint = fmt.Sprintf("%s://%s", access.Protocol, access.Endpoint)
+			break
+		}
+	}
+
+	// If no /init path found, use the first entryPoint endpoint with /init appended
+	if initEndpoint == "" {
+		if len(redisCacheInfo.EntryPoints) > 0 {
+			initEndpoint = fmt.Sprintf("%s://%s",
+				redisCacheInfo.EntryPoints[0].Protocol,
+				redisCacheInfo.EntryPoints[0].Endpoint)
+		} else {
+			respondError(c, http.StatusInternalServerError, "SANDBOX_INIT_FAILED",
```

Comment:
I would recommend by default fallback to using http://ip+ picod port. so making the `EntryPoints` not a must, and mitigate user's configure conplexity